src/pgp_sequoia.h

```diff
@@ -298,8 +298,11 @@ PEP_STATUS pgp_get_key_rating(
  *  @param[in]  changed_key_index    uint64_t *
  *  
  */
-PEP_STATUS pgp_import_keydata(PEP_SESSION session, const char *key_data,
-                              size_t size, identity_list **private_idents,
+PEP_STATUS pgp_import_keydata(PEP_SESSION session,
+                              const char *key_data,
+                              size_t size,
+                              identity_list **idents,
```

Comment:
The parameter is missing in the comment...

Reply:
Cleaned that up, plus added the changelog just in case.